tests/README.md

```diff
@@ -1,26 +1,52 @@
 ## TDX Tests
 
-The tests should be run locally on TDX host.
+This folder contains TDX tests.
 
-It is assumed that these commands are running on a TDX host
-that has been setup properly.
+### Pre-requisites
 
-The guest image must be available at the location : `checkbox-provider-tdx/data/tdx-guest.qcow2`
-The guest image must enable ssh server with password-based authentication for `root` user.
-The root user password must be `123456`
+- The tests must be executed a TDX host that has been setup properly.
 
-Run tests with pytest
+- The guest image must be available at the project folder with the name `tdx-guest.qcow2`
+  You can follow the instructions in the project README to create one guest image.
+  You can specify a path to the guest image with `TDXTEST_IMAGE_FILE` environment variable.
+
+- The guest image must enable ssh server with password-based authentication for `root` user.
+  The root user password must be `123456`
+
+### Run tests with pytest
+
+- Run sanity tests to check the TDX host setup:
+
+```
+$ cd tests
```

Comment:
Perhaps just include `cd tests` once at the top of this section, and then omit from all the examples below.